packages/edit-site/src/components/header/index.js

```diff
@@ -81,62 +82,73 @@ export default function Header( {
 
 	return (
 		<div className="edit-site-header">
-			<MainDashboardButton.Slot>
-				<FullscreenModeClose />
-			</MainDashboardButton.Slot>
-			<div className="edit-site-header__toolbar">
-				<Button
-					isPrimary
-					isPressed={ isInserterOpen }
-					className="edit-site-header-toolbar__inserter-toggle"
-					onClick={ onToggleInserter }
-					icon={ plus }
-					label={ _x(
-						'Add block',
-						'Generic label for block inserter button'
+			<div className="edit-site-header_start">
```

Comment:
The changes in this file add three wrapper divs around the content of the header. One for the "start", "center", and "end". The start and end divs have `flex: 1`. This way, the center div is always centered in the middle of the viewport regardless of the width of the content inside the start/end divs.